news/1191.plugin

```diff
@@ -0,0 +1,3 @@
+patch the Ray Plugin. Changes `--docker` to `--run-env=docker` in `ray exec` calls.
+Fixes `_remote_invoke.py` file path bug to support Docker.
+Removes redundant hidden behavior in `EnvSetupConf`.
```

Comment:
1. news fragments should almost always be a single line. From the user's pov, just say that you fixed docker support in the RayLauncher
2. move the news fragment to the news section in the plugin dir.